app/views/staff/proposals/show.html.haml

```diff
@@ -43,7 +43,6 @@
       = render partial: 'proposals/confirmation_notes', locals: {proposal: proposal, notes: proposal.confirmation_notes}
 
     .col-md-4
```

Comment:
so move the public_tags? check up here since the column and its partial are only there to display tags